tests/backwards_compatibility_tests/requirements.txt

```diff
@@ -1,6 +1,4 @@
 pytest==7.4.3
-pillow==9.3.0
-numpy==1.23.4
 marqo
```

Comment:
we do not specify marqo version here, which means we always use the latest one. Would this cause issue since py-marqo has requirements of minimum version of Marqo server. Can we use newer version of py-marqo to send request to older version of Marqo?

Reply:
discussed offline that I have tested till version 2.9 with latest py-marqo